waterloo-rocketry/website-react
# added jlcpcb to bronze sponsor

## Description

Added JLC to bronze sponsor



This PR closes nothing :(

## Developer Testing

Testing done:
- like literally none. Coding on notepad rn

## Reviewer Testing


Steps to view/test Feature X:
1. `git clone` this repo
1. `npm install`
1. `npm start`
1. go to sponsorship page and see if jlc logo is under bronze and that the link works

Steps to view/test Feature Y:
1. idk tbh. If the website isn't broken it means it works

I'll merge once we get the money lol


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/waterloo-rocketry/website-react/95)
